yarn.lock

```diff
@@ -9377,12 +9377,12 @@ iterate-value@^1.0.2:
     es-get-iterator "^1.0.2"
     iterate-iterator "^1.0.1"
 
-jbox@^1.0.8:
```

Comment:
yarn.lock file probably should not be a part of the change?

Reply:
It's just the opposite. We store the yarn.lock to be sure that we build exactly the same executable always.